packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -321,8 +321,12 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
       }
     );
 
+    const selectedIndex = suggestions.findIndex(suggestion => suggestion.selected);
     if (resultsMaximumNumber) {
-      suggestions = suggestions.slice(0, resultsMaximumNumber);
+      suggestions =
+        selectedIndex >= resultsMaximumNumber
+          ? suggestions.slice(selectedIndex - resultsMaximumNumber + 1, selectedIndex + 1)
+          : suggestions.slice(0, resultsMaximumNumber);
     }
```

Comment:
The output of findIndex()
is bound by the size of the suggestions array. If there is no selected suggestion, it will return -1
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

I was thinking about bounds checking, but the slice() function's 'end index' parameter can be greater than the size of the array.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Reply:
Specifically:

>If end is greater than the length of the sequence, slice extracts through to the end of the sequence (arr.length).